agent/base

```diff
@@ -156,33 +137,25 @@ function get_redhat_version() {
     cat /etc/redhat-release | awk '{ print $7 }'
 }
 
-if [[ -z "${_PBENCH_UNIT_TESTS}" ]]; then
-    function check_enable_copr {
-        local copr_user=$1
-        local copr_name=$2
+function check_enable_copr {
+    local copr_user=$1
+    local copr_name=$2
 
-        if is_fedora ; then
-            dnf copr enable ${copr_user}/${copr_name}
+    if is_fedora ; then
+        dnf copr enable ${copr_user}/${copr_name}
+    else
+        if is_redhat ; then
```

Comment:
Does this cover CentOS as well?

Reply:
Good question, filed #1950 to track separately.